doc/changelog/05-tactic-language/13920-ltac2-ind-api.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  Added an Ltac2 API to manipulate the inductive type
```

Comment:
```suggestion
  Added the Ltac2 API `Ind.v` for manipulating inductive types
```
Make it easier for readers to immediately locate the API.  Also, `Ind` seems a bit cryptic.  Wouldn't it be clearer to call it something like `Inductive`?

Reply:
I think `Ind` is fine, though I don't object to `Inductive`.  This is sort-of a low-level API, so I don't think it hurts too much to have it abbreviated.

@ppedrot, do you want to commit the suggestion to the CHANGES?